datadog.go

```diff
@@ -22,6 +25,18 @@ func init() {
 	log.SetOutput(os.Stderr)
 }
 
+func expandPath(path string) string {
```

Comment:
@abhiyerra That doesn't seem to expand the `~/` shortcut. http://play.golang.org/p/cojcjJJiLt